src/workload.rs

```diff
@@ -309,13 +309,12 @@ impl WorkloadInformation {
     pub async fn fetch_workload(&self, addr: &IpAddr) -> Option<Workload> {
```

Comment:
All those changes in `workload.rs` seem unrelated.

Reply:
These are meant to be commited to https://github.com/istio/ztunnel/pull/262